integrations/jira/.port/resources/blueprints.json

```diff
@@ -14,7 +93,17 @@
         }
       }
     },
-    "calculationProperties": {}
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "board": {
+        "target": "jiraBoard",
+        "title": "Board",
+        "description": "The Jira board that contains this sprint",
```

Comment:
Shouldn't it be `The Jira board that contains this project` since you're on the project blueprint?